spec/spec_helper.rb

```diff
@@ -13,4 +13,5 @@
 begin
   require 'spec_helper_local'
 rescue LoadError
+  puts 'LoadError rescue'
```

Comment:
it'd be good to have something more informative here. maybe something like this?
```ruby
begin
  require 'spec_helper_local'
rescue LoadError => loaderror
  puts "Could not require spec_helper_local: #{loaderror.message}"
end
```

Reply:
Great suggestion Eric, thanks, have added this.